tools/zookeeper/enable_and_wait.py

```diff
@@ -3,10 +3,15 @@
 import os
 import sys
 import time
+from socket import gethostbyname
 from tools.zookeeper import Zookeeper
 
 def is_online(ip):
-  return (os.system(f"ping -c 1 {ip} > /dev/null") == 0)
+  try:
+      addr = gethostbyname(ip)
+      return (os.system(f"ping -c 1 {addr} > /dev/null") == 0)
+  except:
```

Comment:
Mind fixing the bare except? this is causing the static analysis to fail.

Reply:
Thanks for the feedback @robbederks 

I'll do it right away.